pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
     chmod -x $dest/*.ttf
   '';
 
-  outputHashAlgo = "sha256";
-  outputHashMode = "recursive";
-  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
```

Comment:
I dropped it because the build does not actually produce a bit-for-bit reproducible output, or at least not always. (The build failed before the update.)

I can't tell why unpacking an archive and copying files around would not produce the same result, so it would be nice to fix this.